.circleci/scripts/rebase-and-unit-test.sh

```diff
@@ -0,0 +1,22 @@
+
+baseBranch=$1
+echo "Rebasing from branch: $baseBranch"
+
+if ! (git rebase $baseBranch); then
+  echo "Failed to rebase branch $CIRCLE_BRANCH"
+  git rebase --abort
+  exit 1
+fi
+if [ "$isIntegration" = true ]; then
```

Comment:
how is this ever true?

Reply:
This is from the pre-existing script. I just moved this code from a different file to run on the merchant branch rather than the PR branch. Not sure when it would be true

Reply:
Removed this check since we are only running unit tests now